### PR TITLE
Fix broken links and update outdated content in search page

### LINKED ISSUE #221: Provide a site-wide search
A search function for each page would be good.

### PR DESCRIPTION
**Pull request by zeulb**
Fixes #221 
Preview: http://rawgit.com/zeulb/website/221-fix-broken-links-in-search/contents/search.html

Possibly things to do before the search page can be deployed:
- Link with anchor doesn't expand the content anymore in schedule page (Ex: http://nus-cs2103.github.io/website/contents/schedule.html#activity-content-week2)
- Add search contents from other weeks in schedule page

### CODE REVIEW

**Comment by damithc** [comment]
@acjh for your review please. Also look into this issue mentioned by @zeulb 
> Link with anchor doesn't expand the content anymore in schedule page

**Comment by acjh** [comment]
Did an anchor use to expand the content? Doesn't seem like default behaviour.

**Comment by damithc** [comment]
I don't remember having such a functionality. @zeulb do you? 
Even if the feature was there, it could have been forgotten because it's unlikely it was used outside of the search feature, which was not deployed either.
Anyway, it's a good feature to have I guess.

**Comment by zeulb** [comment]
I just checked my old staging and apparently it's not, I guess I remembered this wrongly.
In any case, I think this would be necessary before deploying the search feature.

File: contents/search.html [hunk 1]
```diff
@@ -66,144 +70,133 @@
...
+          <category text="Project Deliverables">
             <category text="V0.0 : The Concept">
-              <keyword text="Techniques to be applied" href="handbook.html#handbook-project-v00"></keyword>
+              <keyword text="Objective" href="handbook.html#handbook-project-v00"></keyword>
               <category text="Sections to include">
                 <keyword text="About Us Page" href="handbook.html#handbook-project-v00"></keyword>
                 <keyword text="User Guide" href="handbook.html#handbook-project-v00"></keyword>
                 <keyword text="User stories" href="handbook.html#handbook-project-v00"></keyword>
                 <keyword text="Non-functional requirements" href="handbook.html#handbook-project-v00"></keyword>
                 <keyword text="Prodcut Survey" href="handbook.html#handbook-project-v00"></keyword>
```
Comment (acjh) [inline contents/search.html]: Should be: *Product* Survey

File: contents/search.html [hunk 2]
```diff
@@ -26,7 +26,7 @@
         </main-category>
         <main-category text="Lectures" type="primary" label="Handbook" href="handbook.html#handbook-lectures">
           <keyword text="Timing/venue" href="handbook.html#handbook-lectures"></keyword>
-          <keyword text="Attendance requirements" href="handbook.html#handbook-lectures"></keyword>
+          <keyword text="Attendance" href="handbook.html#handbook-lectures"></keyword>
           <keyword text="Webcast" href="handbook.html#handbook-lectures"></keyword>
           <category text="Handouts">
             <keyword text="Where to find handouts" href="handbook.html#handbook-lectures"></keyword>
```
Comment (acjh) [inline contents/search.html]: End all questions with `?` for consistency.

File: contents/search.html [hunk 3]
```diff
@@ -66,144 +70,133 @@
             <keyword text="Solution outline" href="handbook.html#handbook-project-product"></keyword>
             <keyword text="Target audience" href="handbook.html#handbook-project-product"></keyword>
           </category>
-          <category text="Product scope">
+          <category text="Product Scope">
             <keyword text="Must-have features" href="handbook.html#handbook-project-scope"></keyword>
-            <category text="Extra features">
+            <category text="Nice to have features">
```
Comment (acjh) [inline contents/search.html]: Should be: *Nice-to-have* features

File: contents/search.html [hunk 4]
```diff
@@ -66,144 +70,133 @@
...
+              <keyword text="Suggested length" href="handbook.html#handbook-project-individualReport"></keyword>
+              <keyword text="Submission" related="TEAMMATES" href="handbook.html#handbook-project-individualReport"></keyword>
+              <keyword text="Deadline" href="handbook.html#handbook-project-individualReport"></keyword>
             </category>
-            <category text="V0.5 Evaluation">
+            <category text="Project Assessment">
               <category text="Product features">
-                <keyword text="Must have features" href="handbook.html#handbook-v05Evaluation"></keyword>
-                <keyword text="Extra features" href="handbook.html#handbook-v05Evaluation"></keyword>
+                <keyword text="Must have features" href="handbook.html#handbook-project-assessment"></keyword>
```
Comment (acjh) [inline contents/search.html]: Should be: *Must-have* features

**Comment by acjh** [comment]
The `p` in [`The product`](https://github.com/nus-cs2103/website/blob/229b73b3b5c89c6f7cfa4be5144a3b7425679d8d/contents/search.html#L68) should also be capitalized for consistency.

**Comment by zeulb** [comment]
Done. Thanks for the review!

File: contents/search.html [hunk 5]
```diff
@@ -61,149 +65,138 @@
...
-                <keyword text="Graded based on qualities of your own code" href="handbook.html#handbook-v05Evaluation"></keyword>
+                <keyword text="Graded based on qualities of your own code" href="handbook.html#handbook-project-assessment"></keyword>
               </category>
               <category text="Testing">
-                <keyword text="How much effort you have invested in the testing process" href="handbook.html#handbook-v05Evaluation"></keyword>
-                <keyword text="How well you have tested the product" href="handbook.html#handbook-v05Evaluation"></keyword>
-                <keyword text="Expectations" href="handbook.html#handbook-v05Evaluation"></keyword>
+                <keyword text="How much effort you have invested in the testing process?" href="handbook.html#handbook-project-assessment"></keyword>
+                <keyword text="How well you have tested the product" href="handbook.html#handbook-project-assessment"></keyword>
+                <keyword text="Expectations" href="handbook.html#handbook-project-assessment"></keyword>
```
Comment (acjh) [inline contents/search.html]: This section in the handbook is now quite different.

**Comment by zeulb** [comment]
Done

**Comment by acjh** [comment]
Looks good. @damithc Any comments?

**Pull request merged**